source/publishing_data/04_configuring_a_source/connectors/kml.rst

```diff
@@ -0,0 +1,27 @@
+KML connector
+=============
+
+The KML File connector is used for KML (.kml) and KMZ (.kmz, compressed KML) files. As defined by `Google <https://developers.google.com/kml/>`_, a KML file is a file used to display geographic data in an Earth browser such as Google Earth.
```

Comment:
I wouldn't use "Earth browser", but only write that "a KML file is a file used to represent geographic data."

You suggest to use Google Earth on the note bellow.